recipe/gen_patch_json.py

```diff
@@ -12,9 +12,20 @@
 import re
 import requests
 import packaging.version
-import pkg_resources
+from packaging.version import parse as parse_version
 
 from get_license_family import get_license_family
+from patch_yaml_utils import (
+    patch_yaml_edit_index,
+    _extract_track_feature,
+    _replace_pin,
+    _rename_dependency,
+    _relax_exact,
+    _pin_stricter,
+    _pin_looser,
+    CB_PIN_REGEX,
+    pad_list,
+)
```

Comment:
Nice. Most of these were originally in this file but then moved to the utils module, right? I am not sure if that module should be named `patch_yaml_utils` though, since its functions are used in other stuff too.

Reply:
My intent is that all patches end up as yaml and so the utility functions are first class and old used in a hacky way in the original code. Thus I think the name (and also the fact that these utils have `_` at the start still) is sensible to me.